### ISSUE TITLE
COBie attribute initial implementation issues

### ISSUE THREAD

**Issue by vdubya** [OpenDBO/OpenRFA#28]
This issue will collect issues/ideas for the COBie attribute implementation for initial discussion and then create separate more specific issues as necessary for each actionable task if there is concurrence, or create a checklist here, whichever seems most appropriate. Attempted to sort by importance.

Overall an extremely good MVP!

## [COBie page](http://dev.openrfa.org/cobie)
* Need column header for first column: “Attribute Name” or “Name”
* Order the Sheet filter list to match COBie standard: Facility, Floor, Space, etc

## [COBie Mapping page](http://dev.openrfa.org/mapping/cobie)
* Need to find a way to address existing built-in Revit parameters such as Manufacturer. Is it possible to obtain the GUID of built-in parameters? Is there a fully itemized list somewhere?
* COBie to IFC mapping would add some unique opportunities. 
     * COBie to IFC mapping data already exists and is part of the standard, so fully vetted and standardized. 
     * Mapping is transitive, so if somebody maps a SharedParameter to an IFC property, and we already have the COBie to IFC property mappings then the COBie to SharedParameter is automatic. Taken to the extreme, linked data might provide the most robust (but complicated) approach.
* Add COBie Attribute to Filter options
* Would be useful if spreadsheet tab name in COBie Attribute column (ie Component) were hyperlinked to trigger a filter by that tab

## Applies to both pages:
* Current lists are specific to COBie v2.4. Need versioning strategy as COBie v3 is not far away. My hunch is separate tables would be best rather than versioning per attribute. 
* Need to update titles/labels to include “v2.4”
* Would like to see way to increase default list display length, up to a very long list, including “All” even if that is time consuming it would be helpful to see all at once. It’s <300 so not too taxing.
* Column headers should be sortable (true for SP pages too)

## Applies to other:
* Recommend including a link to the COBie Attributes page in the main menu within Shared Parameters below Parameter Sets (or something better?).

**Comment by jmerlan** [comment]
Great list. I will start working on it. 

One recommendation moving forward, I'd probably say that each of these should be broken out as a separate issue. It will be easier to track that way.

**Comment by vdubya** [comment]
@jmerlan Agreed on the separate list. I tried to capture everything to discuss first as I tried to mention in the first sentence but maybe better to just start with separate issues from the get go. I liked what you did with the checkboxes but it is much better to have separate issues for tracking etc as you indicate. I will edit and split up.

**Comment by vdubya** [comment]
Separated out individual COBie issues and provided links to them from here. Still need to separate out a few more.
